src/utils/AxiosInstance.ts

```diff
@@ -11,6 +11,7 @@ instance.interceptors.request.use(
     }
     const kcToken = sessionStorage.getItem(SessionStorageKeys.KeyCloakToken)
     request.headers.common['Authorization'] = `Bearer ${kcToken}`
+    request.headers.common['App-Name'] = import.meta.env.APP_NAME
```

Comment:
Sample call to Auth API:
![image](https://github.com/user-attachments/assets/f1b3f998-98ca-4efa-8a8e-f65a7745f0bd)

Sample call to Legal API:
![image](https://github.com/user-attachments/assets/da42662c-d4b8-402a-8ccf-01d53830be9c)